src/components/Mutation.test.tsx

```diff
@@ -0,0 +1,70 @@
+jest.mock('../client', () => {
+  const d = { data: 1234, error: 5678 };
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { delay, fromValue, pipe } = require('wonka');
+  const mock = {
+    executeMutation: jest.fn(() =>
+      pipe(
+        fromValue({ data: 1, error: 2 }),
+        delay(200)
+      )
+    ),
+  };
+
+  return {
+    createClient: () => mock,
+    data: d,
+  };
+});
+
+import * as React from 'react';
+import { act, cleanup, render } from '@testing-library/react';
+import { Mutation } from './Mutation';
+import { createClient } from '../client';
+
+// @ts-ignore
+const client = createClient() as { executeMutation: jest.Mock };
+const query = 'mutation Example { example }';
+
+describe('Mutation', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('Should execute the mutation', done => {
+    let execute = () => {},
+      props = {};
+    const Test = () => <p>Hi</p>;
+    const App = () => {
+      return (
+        // @ts-ignore
```

Comment:
I really can't find why this is needed to be fair, it's complaining about Element | null while I'm returning a React element